components/content/src/front_matter/section.rs

```diff
@@ -68,8 +68,8 @@ pub struct SectionFrontMatter {
     #[serde(skip_serializing)]
     pub aliases: Vec<String>,
     /// Whether to generate a feed for the current section
-    #[serde(skip_serializing)]
-    pub generate_feed: bool,
+    #[serde(skip_serializing, alias = "generate_feed")]
```

Comment:
you don't need it here i think

Reply:
Zola will ignore unknown fields here as well though which again ends up with the "suddenly an option stopped working without an error" problem

Which I'm not sure is solvable within serde